src/pybind/mgr/cephadm/module.py

```diff
@@ -2535,10 +2535,42 @@ def _apply(self, spec: GenericSpec) -> str:
 
         return self._apply_service_spec(cast(ServiceSpec, spec))
 
+    def _get_candidate_hosts(self, placement: PlacementSpec) -> List[str]:
```

Comment:
similar naming concerns to my other comment on `_validate_placement_spec`

Reply:
fixed